gaming/src/golang/item-service/go.sum

```diff
@@ -0,0 +1,1342 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
What is this file? Can we avoid checking this in and dynamically generate it if necessary ?

Reply:
From [best practices](https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions): 

```Ensure your go.sum file is committed along with your go.mod file.```